app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Please modified the views in devise for the registration and login to match the wireframes and their styling.

Your Design | 
:------------:|
<img width="1067" alt="Screen Shot 2022-11-10 at 6 55 45 PM" src="https://user-images.githubusercontent.com/39523312/201182665-e35b5216-7e0b-45c5-aa70-378c3dc38e4f.png"> | 

Wireframe |
:-------------:|
<img width="367" alt="Screen Shot 2022-11-10 at 6 55 24 PM" src="https://user-images.githubusercontent.com/39523312/201182677-e0cf7cab-6020-4d3b-8c85-ce4238eda339.png"> |

Reply:
@mmsesay , thank you for the reviews. Changes updated for both wireframe styling and confirmable action